controllers/argocd/policyrule.go

```diff
@@ -29,7 +29,22 @@ func policyRuleForApplicationController() []v1.PolicyRule {
 }
 
 func policyRuleForRedis(client client.Client) []v1.PolicyRule {
-	rules := []v1.PolicyRule{}
+	rules := []v1.PolicyRule{
+		{
+			APIGroups: []string{
+				"",
+			},
+			Resources: []string{
+				"secrets",
+			},
+			Verbs: []string{
+				"get",
+				"list",
+				"watch",
+				"create",
```

Comment:
what do you think on limiting this only to get? If I am not wrong, Redis will only read the secret and creation will happen in operator manager.

Reply:
Agreed. Made changes accordingly.